firmware/application/external/external.ld

```diff
@@ -55,6 +55,7 @@ MEMORY
     ram_external_app_ook_editor(rwx) : org = 0xADCE0000, len = 32k 
     ram_external_app_remote(rwx) : org = 0xADCF0000, len = 32k 
     ram_external_app_mcu_temperature(rwx) : org = 0xADD00000, len = 32k 
+    ram_external_app_fmradio(rwx) : org = 0xADE00000, len = 32k 
```

Comment:
This part seems a bit incorrect, len = 32K so next address should be 0xADD10000
If you don't mind, I will fix this in one of my PR

Reply:
fixed in #2429 #2431